client/package.json

```diff
@@ -16,7 +16,9 @@
     "@material-ui/icons": "^4.0.0",
     "axios": "^0.19.0",
     "formik": "^1.5.7",
+    "jest-dom": "^3.5.0",
     "jwt-decode": "^2.2.0",
+    "prettier": "^1.18.2",
```

Comment:
Are you sure that prettier should be here?

1. This library is using only for developers, so it should move to devDependencies
2. It should be moved to root directory

Reply:
Good catch with devDependencies, thanks :smile: 
Prettier is needed for creating snapshots and without it jest throws an error

Reply:
he? Could you provide us this error message?

Reply:
`Cannot find module 'prettier' from 'setup_jest_globals.js'`